SPECS/liburing/liburing.spec

```diff
@@ -1,14 +1,13 @@
 Summary:        Linux-native io_uring I/O access library
 Name:           liburing
-Version:        2.0
```

Comment:
Can we get some sort of assurance that the public API hasn't changed between these two versions? The ABI break is somewhat workable, but I'd rather not also start breaking builds randomly for those using this package.